kore/src/Kore/Exec/GraphTraversal.hs

```diff
@@ -420,7 +434,7 @@ instance Pretty a => Pretty (TraversalResult a) where
         GotStuck n as ->
             Pretty.hang 4 . Pretty.vsep $
                 ("Got stuck with queue of " <> Pretty.pretty n)
-                    : map Pretty.pretty as
+                    : map (either Pretty.pretty Pretty.pretty) as
```

Comment:
Maybe better to describe in text whether it was `Left` ("stuck") or `Right` ("vacuous")?